utils/agreement-utils/src/typed_props.rs

```diff
@@ -211,21 +225,34 @@ mod test {
     use super::*;
 
     #[test]
-    fn test_wasm_1() {
+    fn offer_json_test() {
         let offer = OfferDefinition {
             node_info: NodeInfo::with_name("dany"),
             srv_info: ServiceInfo {
                 inf: InfNodeInfo::default().with_mem(5.0).with_storage(50.0),
                 exeunit_info: serde_json::json!({"wasm.wasi.version@v".to_string(): "0.9.0".to_string()}),
                 multi_activity: false,
+                payload_manifest: true,
             },
             com_info: Default::default(),
             offer: OfferTemplate::default(),
         };
 
-        eprintln!(
```

Comment:
I had no idea what this test did. It was supposed to print to std:err only on error? Is something checking when test is executed?
I converted it to some rudimentary check of serialized offer because we have hardcoded str literals so it may at least prevent some typos.